store/models.py

```diff
@@ -30,3 +30,11 @@ def __str__(self):
         else:
             return f'{self.book.title} - Available'
 
+
+class RatingOfBook(models.Model):
+    book = models.ForeignKey(Book, on_delete=models.CASCADE)
+    ratingValue = models.PositiveIntegerField(null=True, blank=True)
```

Comment:
You can also use `MinValueValidator` and `MaxValueValidators` here to have database level constraints set on this field.